gems/aws-sdk-core/spec/aws-sdk-core/middleware/checksums_spec.rb

```diff
@@ -0,0 +1,174 @@
+# frozen_string_literal: true
+
+require_relative '../../spec_helper'
+
+module AWS::SDK::Core
+  module Middleware
+    Input = ::Struct.new(
+      :request_algorithm,
+      :request_validation
+    ) do
+      include Hearth::Structure
+    end
+
+    describe Checksum do
```

Comment:
Do we get full coverage with these specs? (check output)

Reply:
Not quite - The streamed signing is not implemented so isn't covered.